coding-concepts.js

```diff
@@ -20,24 +20,24 @@ const greeter = (name) => {
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "LEARN Student"
+// b) Verify and explain: undefined, in order for "LEARN Student" to be displayed using string interpolation the string will need to be set to a variable. Then when you console log you would console.log(greeter(variableName)) for it to work.
```

Comment:
Look at the function a little closer. What is missing from the code block? It is crucial for the function to display data.